core/src/test/java/com/netease/arctic/utils/JacksonUtilsTest.java

```diff
@@ -0,0 +1,196 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netease.arctic.utils;
+
+import static org.junit.Assert.*;
+
+import org.apache.paimon.shade.jackson2.com.fasterxml.jackson.databind.JsonNode;
+import org.junit.Test;
+
+import java.util.*;
+
+public class JacksonUtilsTest {
+
+  @Test
+  public void jacksonTest() {
+
+    JsonTestBean testObject = new JsonTestBean();
+    testObject.setBoolValue(true);
+    testObject.setIntValue(3);
+    testObject.setStringValue("abcd");
+    Map<String, String> map = new HashMap<>();
+    map.put("a", "b");
+    testObject.setMapValue(map);
+
+    String jasonString = JacksonUtils.toJSONString(testObject);
+
+    String expectedValue =
+        "{\"stringValue\":\"abcd\",\"intValue\":3,\"boolValue\":true,\"mapValue\":{\"a\":\"b\"}}";
+    assertEquals(expectedValue, jasonString);
+
+    JsonTestBean objReadFromJson = JacksonUtils.parseObject(jasonString, JsonTestBean.class);
+    assertEquals(testObject, objReadFromJson);
+
+    JsonNode jsonNode = JacksonUtils.fromObjects(objReadFromJson);
+
+    Integer expectedInt = 3;
+    assertEquals(expectedInt, JacksonUtils.getInteger(jsonNode, "intValue"));
+    // return null if key does not exist
+    assertNull(JacksonUtils.getInteger(jsonNode, "NoIntValue"));
+
+    String expectedString = "abcd";
+    assertEquals(expectedString, JacksonUtils.getString(jsonNode, "stringValue"));
+    // return null if key does not exist
+    assertNull(JacksonUtils.getString(jsonNode, "NoStringValue"));
+    // return default value if key does not exist, and default value passed
+    assertEquals("DefaultValue", JacksonUtils.getString(jsonNode, "NoStringValue", "DefaultValue"));
+
+    assertTrue(JacksonUtils.getBoolean(jsonNode, "boolValue"));
+    // returns false if key does not exist
+    assertFalse(JacksonUtils.getBoolean(jsonNode, "NboolValue"));
+    assertTrue(JacksonUtils.getBoolean(jsonNode, "NboolValue", true));
+
+    Map<String, Object> yamlMap = new HashMap<>();
+    Map<String, Object> amsNode = new HashMap<>();
+    amsNode.put("arctic.ams.server-host.prefix", "127.");
```

Comment:
We shouldn't use the old name `arctic`, this will be import additional renaming work later.